pkgs/data/icons/la-capitaine-icon-theme/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "la-capitaine-icon-theme";
```

Comment:
```suggestion
  pname = "la-capitaine-icon-theme";
```

Reply:
Should i always use `pname` instead of `name` in all other new packages i will create?

Reply:
I'd say where appropriate. For most packages you'll need the value of `pname` throughout the expression.